cli/ssh.go

```diff
@@ -25,19 +25,18 @@ import (
 	"golang.org/x/xerrors"
 	"gvisor.dev/gvisor/pkg/tcpip/adapters/gonet"
 
-	"github.com/coder/retry"
-	"github.com/coder/serpent"
-
 	"cdr.dev/slog"
 	"cdr.dev/slog/sloggers/sloghuman"
-
 	"github.com/coder/coder/v2/cli/cliui"
 	"github.com/coder/coder/v2/cli/cliutil"
+	"github.com/coder/coder/v2/cli/xterminal"
```

Comment:
This `x` convention feels unfamiliar in this repo, thoughts about renaming it? Going with `cliterm` would certainly match the convention in this package.

Reply:
Moving to pty as per Kyle's suggestion